extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss

```diff
@@ -15,17 +16,18 @@ $close-button-background: '/resources/wikia/ui_components/modal/images/close-dar
 	background: url('/extensions/wikia/PotentialMemberPageExperiments/images/858-px-background.png') no-repeat center bottom;
 	background-size: cover;
 	border-bottom: 2px solid $color-blue;
-	height: 150px;
```

Comment:
I'm not sure is it safe to remove this property. Right now banner in article is smaller and has cut off background